test-fixtures/integrations/withImports.ts

```diff
@@ -0,0 +1,66 @@
+import { BrowserTracing, Integrations } from '@sentry/browser';
```

Comment:
I don't remember if we supported this previously anywhere but theoretically, users could also just `import { BrowserTracing as MyBrowserTracing } from '@sentry/browser`.

In case we currently don't handle this, I think it's also totally fair to ignore this for now. (Even if we say we want to handle this, feel free to open a follow up PR).

Reply:
yeah, this will not work with renamed imports for now! We can look at this later if needed :)